cli/src/print.rs

```diff
@@ -0,0 +1,244 @@
+use {
+    comfy_table::{modifiers::UTF8_ROUND_CORNERS, presets::UTF8_BORDERS_ONLY, Row, Table},
+    gluesql::prelude::{Payload, PayloadVariable},
+    std::io::{Result, Write},
+};
+
+pub struct Print<W> {
+    pub output: W,
+}
+
+impl<W: Write> Print<W> {
+    pub fn new(output: W) -> Self {
+        Print { output }
+    }
+
+    pub fn payload(&mut self, payload: Payload) -> Result<()> {
+        let mut affected = |n: usize, msg: &str| -> Result<()> {
+            writeln!(
+                self.output,
+                "{} row{} {}\n",
+                n,
+                if n > 1 { "s" } else { "" },
+                msg
+            )
+        };
+
+        match payload {
```

Comment:
[nit]
Is it possible to print something when the sql is `CREATE TABLE ~~`?

Reply:
Sure, do you have any suggestion on it?